Dockerfile

```diff
@@ -3,19 +3,24 @@
 #
 # SPDX-License-Identifier: Apache-2.0 AND AFL-3.0
 
-# We can't use alpine anymore because crypto has rust deps.
-FROM node:20-slim
+FROM alpine/git:latest as git-stamp-stage
```

Comment:
is it worth doing this or should i just install git in the base image?

Reply:
I would argue the download overhead of the extra image is larger than an apt install which can be cached in a layer. The size of the whole image is probably larger than a single layer :thinking:

Reply:
oki